release_docs/INSTALL_CMake.txt

```diff
@@ -27,11 +27,11 @@ Obtaining HDF5 source code
    2. Obtain compressed (*.tar or *.zip) HDF5 source from
          https://portal.hdfgroup.org/display/support/Building+HDF5+with+CMake
       and put it in "myhdfstuff".
-      Uncompress the file. There should be a hdf5-1.13."X" folder.
+      Uncompress the file. There should be a hdf5-1.15."X" folder.
```

Comment:
Another 1.15(1.13) should be 1.14

Reply:
Holy crap this file needs to be fixed so we don't have to update version numbers

Reply:
Fixed in next PR